contrib/nitiwiki/README.md

```diff
@@ -128,6 +128,50 @@ For more details on each option see `WikiConfig`.
 
 To add content in your wiki simply add markdown files (with `.md` extension) into the `pages/` folder.
 
+#### Write content
+
+Nitiwiki only support content written in
+[markdown format](http://daringfireball.net/projects/markdown/).
+
+#### Link wiki articles
+
+In nitiwiki, linking to section or article is done using the *wikilinks*.
+
+WikiLinks provide easy linking between pages of the wiki.
+To create a wikiLink, just put the name of the page to link to in double brackets.
```

Comment:
You use both wikiLink and *wikilink* (in the previous paragraph). And why is it a single word, is it a trademark or something like that?